### ISSUE TITLE
294 vids and stills to upload from cur50 folder

### ISSUE THREAD

**Issue by blbrunner** [membean/content#178]
294 new mostly MS4 vids and stills are ready to go .... all names of clips and stills have been checked, and everything else on my end is done for these, including all sources and captions. If you could please get these up ASAP that would be great--questions are all written for them and these will need to go up before questions can be uploaded.  These will be the last vids to upload until I start on the MS5 vids around the beginning of Septermber.

**Comment by MicahElliott** [comment]
I think these are done now.  Please check a few for me.  I don't see stills yet for MS5 so I'll wait for a new ticket to handle those.

**Comment by blbrunner** [comment]
Hey Micah--
   Yes, I am just starting to work on the MS5 vids now.  Shall I send them
to you in sets of 50 or 100?  Either way is fine with me ... although 100
might be easier so you don't have to do so many uploads.  I'll check on
them and let you know!

Brett

**Comment by blbrunner** [comment]
hmm ... I'm not sure these are done.  these didn't upload correctly:
 auxiliary, catastrophe, civil, deadlock (these were the first four I
checked and they did not change from before).  Not sure why; there were
vids already there, so they should have been replaced with the new ones.
Not sure why ... also, the word "dissonance" was not put in, which did not
have one there before either, so I think that these all just didn't get in
there.

please let me know when these have been uploaded and I'll check again.
 thanks!  Brett

**Comment by blbrunner** [comment]
yeah, but that's the wrong vid.  it should be from a trucker company ....
look at the vid in the cur50 folder for that.  you will see that it is
different, and that the current source/caption does not match the vid.

**Comment by MicahElliott** [comment]
Should be done now. Please spot check. Probably better to wait till tomorrow since some existing media were replaced.

**Comment by blbrunner** [comment]
OK.  Will do.  And thanks!  Brett

**Comment by blbrunner** [comment]
Ok, this is now tomorrow so I'm going to check.  Wll let you know.  Brett

**Comment by blbrunner** [comment]
Micah--
    The still for "deadlock" didn't make it, but the clip did.  Not sure
why.  Is there a way to check out the stills going in?  Maybe because there
was one up there prior to it?  The same thing happened with "arrogant,"
which also had a clip up beforehand .. but all the other ones I checked
were fine ... so I think that previous stills were not replaced for
whatever reason, but the clips were.  So those just need to be changed.
Other than that, all is well.

Brett
